LICENCE

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+Copyright (C) 2017 Crown Copyright (Government Digital Service)
```

Comment:
2018?

Reply:
See https://gds-way.cloudapps.digital/manuals/licensing.html#licensing: "The year should be the year the code was first published. "

Reply:
👍 Cool beans.